src/data/mainnet/optimism-tokens-info.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "name": "Ether",
+    "symbol": "ETH",
+    "decimals": 18,
+    "isNative": true,
+    "isL2Native": true,
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/ETH.png",
+    "showZeroBalance": true,
```

Comment:
should this be false? I guess we only have cash in eligible tokens on the zero state so the Buy button shows up on asset details

Reply:
changed it to false